docs/topics/ref_new-features-7-1-0.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.1.0/master.adoc
+
+
+:_newdoc-version: 2.18.2
+:_template-generated: 2024-07-01
+
+:_mod-docs-content-type: REFERENCE
+
+[id="new-features-7-1-0_{context}"]
+= New features
+
+This section provides the new features and improvements of the {ProductFullName} 7.1.0.
+
+.A support for analyzing applications managed with Gradle has been added to {ProductShortName} 
+
+Previously, you could use the {ProductFullName} to analyze Java applications managed only with Maven. With this update, {ProductShortName} can also extract dependencies from Gradle projects. As a result, you can now analyze applications that use Gradle instead of Maven. 
+
+
+
+.Support for analyzing .NET applications has been added to MTA
+
+In {ProductFullName} 7.1.0, you can use MTA to analyze Windows-only .NET framework applications to aid migration from version 4.12 or later to multi-platform .NET 8.0 running on OpenShift Container platform.
+
+
+
+.Assessment and review statuses are now displayed during the archetype assessment and review
+
+With this update, the status of the archetype assessment and review processes is displayed in the {ProductFullName} user interface (UI):
+
+* The assessment statuses are the following:
+** Completed: All required assessments completed.
+** InProgress: The assessment process is in progress.
+** NotStarted: The assessment process has not been started.
+* The review statuses are the following:
+** Completed: A review exists.
+** NotStarted: The review process has not been started.
+
+
+
+.Assessment and review statuses are now displayed in the MTA UI
```

Comment:
This and the previous point are exactly same . Anyone is required.

Reply:
I'm sorry, I'm a bit confused here. What do you mean by "the same"?